code/game/sound.dm

```diff
@@ -45,6 +45,9 @@
 	S.channel = channel || SSsounds.random_available_channel()
 	S.volume = vol
 
+	if(channel)
+		S.volume *= client.prefs.get_channel_volume(channel)
```

Comment:
Most of the affected channels are directly sent to the client, but it doesn't hurt to have this